### DISCUSSION TITLE
Browser freezing with 2023.4.2

### DISCUSSION THREAD

**Discussion by homeontherocks** [tesla-android/issue-tracker#218] [General]
Hello,
Sorry if this has been addressed but I can start up everything fine (including using CarPlay with the Carlinkit adapter) but then after awhile the browser becomes unresponsive.  Only a reboot of the Tesla will fix the problem.  Not sure if this is an ongoing issue or an easy fix but thought I would report.
Thanks!

**Comment by mikegapinski** [comment]
It's fixed in the latest pre-release you can find on Reddit. It should be released widely soon
